stack.yaml

```diff
@@ -75,6 +75,79 @@ extra-deps:
 - git: https://github.com/michaelpj/slack-web.git
   commit: 6ff88895c5da593e7bd09a46d7002441817aa6f8
   # nix-sha256: 1115jngzx2hbvznhnrnm4qs9vn6hsjklxwiwa5i11npr2kxd5rm2
+
+## Node protocols
+- canonical-json-0.6.0.0@sha256:9021f435ccb884a3b4c55bcc6b50eb19d5fc3cc3f29d5fcbdef016f5bbae23a2,3488
```

Comment:
Can we make this its own `extra-deps` stanza so we actually have a self-contained set of things for the node protocols? (i.e. do multiple `extra-deps` stanzas concatenate?)

Reply:
They do not :(

Reply:
Would you rather I make a delimited section in the existing extra-deps for the Node protocols? It's not much, but ... it's almost something (?)

Reply:
Isn't that what it is already (just trailing down the end)?

I think it's fine. Maybe just add a few more comments in the files about what's in here and why and when we're going to remove it.